README.md

```diff
@@ -127,34 +127,31 @@ These numbers are based on [scalability tests](https://github.com/kubernetes/kub
-Kubernetes, kube-state-metrics is focused on generating completely new metrics
-from Kubernetes' object state (e.g. metrics based on deployments, replica sets,
-etc.). The reason not to extend [metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster) with kube-state-metrics' abilities is
-because the concerns are fundamentally different: [metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster) only needs to fetch,
-format and forward metrics that already exist, in particular from Kubernetes
-components, and write them into sinks, which are the actual monitoring
-systems. kube-state-metrics, in contrast, holds an entire snapshot of
-Kubernetes state in memory and continuously generates new metrics based off of
-it but has no responsibility for exporting its metrics anywhere.
+### kube-state-metrics vs. metrics-server
+
+The [metrics-server](https://github.com/kubernetes-incubator/metrics-server)
+is a project that has been inspired by
+[Heapster](https://github.com/kubernetes-retired/heapster) and is implemented
+to serve the goals of the [Kubernetes Monitoring
+Pipeline](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/monitoring_architecture.md).
+It is a cluster level component which periodically scrapes metrics from all
+Kubernetes nodes served by Kubelet through Summary API. The metrics are
+aggregated, stored in memory and served in [Metrics API
+format](https://git.k8s.io/metrics/pkg/apis/metrics/v1alpha1/types.go). The
+metric-server stores the latest values only and is not responsible for
+forwarding metrics to third-party destinations.
+
+kube-state-metrics is focused on generating completely new metrics from
+Kubernetes' object state (e.g. metrics based on deployments, replica sets,
+etc.). It holds an entire snapshot of Kubernetes state in memory and
+continuously generates new metrics based off of it. And just like the
+metric-server it too is not responsibile for exporting its metrics anywhere.
 
 In other words, kube-state-metrics itself is designed to be another source for
-[metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster) (although this is not currently the case).
-
-Additionally, some monitoring systems such as Prometheus do not use [metrics-server](https://github.com/kubernetes-incubator/metrics-server)(Heapster)
-for metric collection at all and instead implement their own, but
-[Prometheus can scrape metrics from metrics-server(Heapster) itself to alert on metrics-server(Heapster)'s health](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
-Having kube-state-metrics as a separate project enables access to these metrics
-from those monitoring systems.
+[metrics-server](https://github.com/kubernetes-incubator/metrics-server)
```

Comment:
I doubt this will ever be the case, let's just remove this sentence, the second sentence of this paragraph is the essence of kube-state-metrics' existence.

Reply:
Done.